tools/cldr-apps/src/main/java/org/unicode/cldr/web/CompareCLDRFile.java

```diff
@@ -22,7 +22,7 @@ private Entry(CLDRFile f, String t) {
             this.file = f;
             this.title = t;
 
-            for (String s : f) {
+            for (String s : f.iterableDefault()) {
```

Comment:
This is the typical situation. Since CLDRFile no longer implements Iterable, it's necessary to call a method on it to get an iterable.